saleor/shipping/models.py

```diff
@@ -203,6 +204,13 @@ class ShippingMethod(ModelWithMetadata):
     maximum_delivery_days = models.PositiveIntegerField(null=True, blank=True)
     minimum_delivery_days = models.PositiveIntegerField(null=True, blank=True)
     description = SanitizedJSONField(blank=True, null=True, sanitizer=clean_editor_js)
+    tax_class = models.ForeignKey(
+        TaxClass,
+        related_name="shipping_methods",
+        blank=True,
+        null=True,
+        on_delete=models.SET_NULL,
+    )
```

Comment:
What do you think about it? 
```
class TaxClass(models.Model):
    <fields>

    @classmethod
    def get_default_pk(cls):
        tex_class, _ = cls.objects.get_or_create(...)
        return tex_class.pk


class Products(models.Model):
    exam_taken = models.ForeignKey(
        ...
        on_delete=models.SET_DEFAULT,
        default=TaxClass.get_default_pk
    )
 ```

Reply:
Refactored models to use this approach, thanks 👍